news/93e1ab65.internal

```diff
@@ -0,0 +1,2 @@
+Update configuration files.
```

Comment:
@gforcada is the name of the file autogenerated or where does 93e1ab65 come from?

Reply:
Yes, autogenerated, a commit hash, or directly something random, no idea, but somewhere in `plone/meta` has the logic for that.

We can modify that to something else if you want